CHANGELOG.md

```diff
@@ -13,7 +13,7 @@
 * Checks that titles and narrative text are at least 50% alpha characters.
 * Restricts titles to a maximum word length. Adds a `UNSTRUCTURED_TITLE_MAX_WORD_LENGTH`
   environment variable for controlling the max number of words in a title.
-
+* Adding a feature to customizing the color scheme of sphinx documentation.
```

Comment:
Don't need a change log entry for this one since the source code didn't change.

Reply:
This has been removed